ui/imports/shared/status/StatusChatInput.qml

```diff
@@ -381,9 +377,9 @@ Rectangle {
 
     function onKeyPress(event) {
         // get text without HTML formatting
-        const messageLength = messageInputField.getText(0, messageInputField.length).length;
+        const messageLength = messageInputField.length
```

Comment:
Does this only count the text as well. We wouldn't want to be counting the HTML tags

Reply:
Yup, see https://doc.qt.io/archives/qt-5.15/qml-qtquick-textedit.html#length-prop:

> Returns the total number of plain text characters in the [TextEdit](https://doc.qt.io/archives/qt-5.15/qml-qtquick-textedit.html) item.

> As this number doesn't include any formatting markup it may not be the same as the length of the string returned by the [text](https://doc.qt.io/archives/qt-5.15/qml-qtquick-textedit.html#text-prop) property.